webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -91,8 +91,12 @@ export const LinkTooltip = ({href, connected}) => {
                     <div className='header mb-1'>
                         <a
                             title={data.repo}
-                            href={href}
-                        >{data.repo}</a>
+                            href={data.base.repo.html_url}
```

Comment:
That is a bit of a hack, but I'm 90% sure it's works. `base` will always belong to the repo the PR was submitted against. Hence, it's fine to use it's `html_url`. Otherwise there would have to be another API call, which I want to avoid.